spec/workers/pages/bust_cache_worker_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+class NewPageBuster
```

Comment:
Feeling like this might not be a good practice since the constant definition leaks to other tests. I am only following the suit with the article one, but I think we should fix this in a separate issue.

Reply:
I agree, but we can keep them there for now :)